vertical-pod-autoscaler/pkg/recommender-external/input/annotations_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package input
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
+)
+
+func TestGetVpaExternalMetrics(t *testing.T) {
+	type args struct {
+		annotations map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
```

Comment:
/nit since we have only one arg, you can put it directly here, that would save couple of lines.
```
tests := []struct {
		name string
		annotations map[string]string
		...
		}
```

I know that the generated templates comes with `fields` and `args` but when cardinality is low in the elements it is better to unfold it.

Reply:
good idea